### DISCUSSION TITLE
New version of closed xml is not having public key(strong name assembly) and digitally not signed

### DISCUSSION THREAD

**Discussion by ZubairImran** [ClosedXML/ClosedXML#2478] [Q&A] [answered]
Hi Team,
Just noticed the concern mentioned in Title(digital signature and strongname). Is this missed?

**Comment by jahav** [comment, marked as answer]
Added dependency wasn't strongly named. Tracked in https://github.com/ClosedXML/ClosedXML/issues/2423

**Reply by ZubairImran** [reply]
Thanks @jahav.
